src/components/IconHelp/IconHelp.unit.spec.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactTestUtils from 'react-dom/test-utils';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
```

Comment:
In general, we prefer new tests to be written using [`@testing-library/react`](https://testing-library.com/docs/react-testing-library/intro/). That's not written down anywhere, so I'm not gonna reject the PR for it or anything, but that library provides a lot more assurances that what we expect to happen actually is.

Reply:
Good to know, thanks Chris. I'll use `@testing-library/react` going forward.